### PR TITLE
Feature/anonymous profile

### PR DESCRIPTION
**Pull request by julka**
Using a consistent anonymous profile id allows AddThis support to migrate analytics to a normal ra-\* profile id if a user ever decides to sign up for an AddThis.com account for analytics or additional tools.

### CODE REVIEW

File: classes/AddThis.php [hunk 1]
```diff
@@ -251,6 +253,39 @@ public function getProfileId() {
...
+      $profile_id = check_plain(AddThis::setAnonymousProfileId());
+    }
+
+    return $profile_id;
+  }
+
+  private function setAnonymousProfileId() {
+    $prefix = AddThis::ANONYMOUS_PROFILE_ID_PREFIX;
+
+    Global $base_url;
```
Comment (matglas) [inline classes/AddThis.php]: Could you make this a small G?

**Comment by matglas** [comment]
Can you create a issue on d.o. (Drupal.org) for others to review?

**Comment by matglas** [comment]
Hi @julka,
Did you see my request?

**Comment by julka** [comment]
Hi @matglas. Yes, I did. :-) I'm sorry, I've been very distracted this last week with other project. I will try to get to this soon.

**Comment by julka** [comment]
I fixed that Global to global, and made a d.o. issue here: https://www.drupal.org/node/2528510

**Pull request closed without merging**